src/main/kotlin/coffeebot/processor/Online.kt

```diff
@@ -8,10 +8,10 @@ import discord4j.core.DiscordClientBuilder
 import discord4j.core.event.domain.lifecycle.ReadyEvent
 import discord4j.core.event.domain.message.MessageCreateEvent
 
-class Online(private val token: String) : MessageProcessor {
+class Online(private val token: String, miltonToken: String?) : MessageProcessor {
```

Comment:
Keep naming consistent. Here, it's miltonToken, while in offline, it's miltonSecret